app/util/ThumbnailGenerator.scala

```diff
@@ -42,6 +42,23 @@ case class ThumbnailGenerator(logoFile: File) {
     graphics.drawImage(bgImage, 0, 0, null)
     Logger.info(s"Creating branded thumbnail for atom $atomId. Image dims ${bgImage.getWidth()} x ${bgImage.getHeight()}. Logo dims ${logoWidth.toInt} x ${logoHeight.toInt} (x:$logoX, y:$logoY)")
     graphics.drawImage(logo, logoX, logoY, logoWidth.toInt, logoHeight.toInt, null)
+
+    if(withBorder) {
+      val borderThickness = 40
+      val hexColour = "#C70000"
+      val colour = Color.decode(hexColour)
+
+      Logger.info(s"Adding a ${borderThickness}px $hexColour border to thumbnail for atom $atomId")
+
+      val currentStroke = graphics.getStroke
+
+      graphics.setStroke(new BasicStroke(borderThickness))
+      graphics.setColor(colour)
+      graphics.drawRect(0, 0, bgImage.getWidth, bgImage.getHeight)
```

Comment:
As it stands this seems to draw a 20px border - presumably as the stroke is centred on the very edge of the image. So we could either do `new BasicStroke(borderThickness * 2)` or 
```
vall borderOffset = borderThickness / 2
graphics.drawRect(borderOffset, borderOffset, bgImage.getWidth - borderOffset, bgImage.getHeight - borderOffset)
```

Both are pretty nasty but the second is slightly less of a hack.

Reply:
Updated:

![image](https://user-images.githubusercontent.com/836140/81271898-9dedba80-9044-11ea-83dc-437c40166064.png)